about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Developer Hackerton</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/mobile.css" media="screen and (max-width: 768px)">
+    <link rel="stylesheet" href="assets/css/desktop.css" media="screen and (min-width: 768px)">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="bar-section">
+        <nav class="top-header">
+            <div class="top-wrapper">
+                <ul class="link-rx">
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-facebook-square"></i></a></li>
+                </ul>
+                <ul class="link-info">
+                    <li>English</li>
+                    <li>Sign Up</li>
+                    <li>Login</li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="nav-bar">
+            <div class="logo">
+                <a href="index.html#welcome" tabindex="1">
+                    <img src="./assets/images/conference_logo.png" alt="Conference Organizers Logo" class="conference-logo logo-effect"/>
+                </a>
+            </div>
+            <div class="myMenu">
+                <ul class="nav-list">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="index.html#program">Program</a></li>
+                    <li><a href="index.html#speakers">Speakers</a></li>
+                    <li><a href="index.html#partners" >Partners</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a class="btn btn-ticket" href="#!">Buy Tickets now</a></li>
```

Comment:
- [ ] Could you, please, add a button that takes one back to the 'Home' page (Top of the page) when a user opens the menu from the about page on mobile view? I believe this will enhance the User Experience greatly. You can change the 'About' to 'Home' in the 'About' page and change it back to 'About' in the 'Home' page of the mobile view.
![image](https://user-images.githubusercontent.com/45668340/146757652-3fbf3623-f863-46df-a4fb-6e27929c5a1b.png)

Reply:
Thank you for pointing this out, i have resolved this